src/cpp/xmlparser/XMLParser.cpp

```diff
@@ -2540,8 +2540,10 @@ XMLP_ret XMLParser::loadXML(const std::string& filename, up_base_node_t& root)
     }
 
     tinyxml2::XMLDocument xmlDoc;
-    if (tinyxml2::XMLError::XML_SUCCESS != xmlDoc.LoadFile(filename.c_str()))
+    tinyxml2::XMLError load_error = xmlDoc.LoadFile(filename.c_str());
+    if (tinyxml2::XMLError::XML_SUCCESS != load_error)
     {
+        logError(XMLPARSER, "Error loading '" << filename << "'. Error code: " << load_error);
```

Comment:
Perhaps you could add more info here and say that next we'll try to load the default profile